TACOCAT.py

```diff
@@ -26,6 +28,8 @@
 #Read in parameters
 Fuel_Type = TCinput.Fuel_Type
 Coolant_Type = TCinput.Coolant
+Geometry_Type = TCinput.Geometry
+
 Hc = TCinput.Hc
 HotF = TCinput.HotF
 

```

Comment:
The geometry core information contained in TACOCAT.py needs to be removed in a future pull request. Especially since this is now contained in the geometry files.